README.md

```diff
@@ -209,9 +216,33 @@ Those files are located in the /built-flexy-java-app/ directory of the release z
 At startup of the Flexy Java application component, it checks for a configuration file in the /usr/ directory of the Ewon. If one is not present, the application will automatically create one with default values.
 
 After the Flexy Java application component has been installed and started, the configuration should be modified to contain the proper values for AppKey and Thingworx Full URL for your Thingworx instance.
-### Thingworx Full URL 
+
+![Configuration Example](https://github.com/hms-networks/flexy-thingworx-connector/blob/local-write-back/images/ExampleConfig.PNG?raw=true)
+
+#### Thingworx Full URL
 This parameter in the config file should be changed to the full URL for the target Thingworx REST API. Users should pay close attention to the scheme designated in the URL. Specifying the scheme "https" will ensure all traffic to the server is encrypted over TLS. For example, setting the ThingworxFullUrl parameter to "https://example.com:8443/Thingworx/Things/ConnectorHost/Services/TakeInfo" will create a connection to host "example.com" on port 8443 using TLS/SSL. It's possible to specify the IP address of the Thingworx instance instead of a domain name. This configuration will assume "http" scheme and communication will not be encrypted. 
 
+#### App Key
+This should match the app key for your Thingworx instance.
+
+#### Log Level
+The Thingworx connector uses the HMS Solution Center logging library for application logging to the Ewon Flexy's realtime logs. See [Log Level](https://github.com/hms-networks/sc-flexy-logger-lib#log-level) for more information.
+
+#### FTP User
+The username for accessing Ewon Flexy via FTP. This user account must be configured for non UTC time zones to be used on the Flexy.
+
+#### FTP Password
+The password for accessing Ewon Flexy via FTP. This user account must be configured for non UTC time zones to be used on the Flexy.
+
```

Comment:
If the intention is that the user will follow the readme for installation instructions we should provide more detail about the FTP user such as what permissions it needs. This would probably be another section in the readme and could be addressed in another PR.

Reply:
I will include a new section on setting up the FTP user in the next PR.